### DISCUSSION TITLE
4월 1주차 refactoring

### DISCUSSION THREAD

**Discussion by punchdrunkard** [dnd-side-project/dnd-8th-1-frontend#336] [01. Refactoring]
.

**Comment by punchdrunkard** [comment]
스프린트 wiki 에 넣을지말지

**Reply by punchdrunkard** [reply]
보류

**Comment by punchdrunkard** [comment]
발견된 버그들

**Reply by punchdrunkard** [reply]
- [x] 만나기 작성 페이지 로그인하지 않으면 접근 제어 (다은)
- #341

**Reply by punchdrunkard** [reply]
- [x] 만나기 작성 페이지 title 설정 (다은)
- #343

**Reply by punchdrunkard** [reply]
meet 메인 페이지에서 페이지 이동 후 필터 선택하면 에러뜸 (다은)
- [x] x 누르면 페이지도 초기화되게 바꾸기
- #345

**Comment by punchdrunkard** [comment]
캘린더 쪽 리팩터링 (진영)

**Comment by punchdrunkard** [comment]
proxy로 로그인 로직 정비하기

**Comment by punchdrunkard** [comment]
기술 문서 두 개 작성하기

**Comment by punchdrunkard** [comment]
Relase Note 다음 스프린트때부터 어떤 내용 개선했는지 정도 적어보기

**Reply by jinyoung234** [reply]
1. [활동 내역 페이지 내 후기 탭 empty UI 및 review 안 보이는 버그 핫 픽스](https://github.com/dnd-side-project/dnd-8th-1-frontend/commit/de3a5ce9c4cadcc1a35a02a40fd9a70f3567b252)
2. [공연 정보 메인 페이지 내 배너 이미지 안 보이는 현상 및 텍스트 그라디언트 수정](https://github.com/dnd-side-project/dnd-8th-1-frontend/commit/4eba8ed651e343a80287e7d62dc9d56501caf1a5)
3. [프로필 등록 하지 않은 유저가 플로팅 버튼 클릭 시 게시물 등록 페이지로 이동을 제한 하는 기능 추가](https://github.com/dnd-side-project/dnd-8th-1-frontend/commit/b0821a42c68e2d58345a4a8d3956c488fe1538f4)

**Reply by punchdrunkard** [reply]
![image](https://user-images.githubusercontent.com/74234333/230721031-a91df1a8-7ffc-48b2-94bb-a58b9271e89f.png)

**Reply by punchdrunkard** [reply]
캘린더쪽 바꾼거 릴리즈노트 에 추가

**Comment by punchdrunkard** [comment]
wiki에 convention 노션 링크 wiki pages로 마이그레이션
